vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps

```diff
@@ -284,3 +284,7 @@ When I wait until element located by `<locator>` contains text `<text>'
 Composite: Then element located '$locator' disappears in '$timeout'
 !-- WARNING: The step "Then element located '$locator' disappears in '$timeout'" is deprecated and will be removed in VIVIDUS 0.6.0
 Then element located by `<locator>` disappears in `<timeout>`
+
+Composite: When I go to the relative URL '$relativeURL'
```

Comment:
we need to have tests for both new and deprecated steps, so please either add new test or change some existing one to check behaviour of the new step

Reply:
added